lib/MusicBrainz/Server/ControllerUtils/JSON.pm

```diff
@@ -0,0 +1,32 @@
+package MusicBrainz::Server::ControllerUtils::JSON;
+
+use base 'Exporter';
+
+our @EXPORT_OK = qw(
+    serialize_pager
+);
+
+sub serialize_pager {
```

Comment:
Wasn't sure of a good place to put this, so I created a new package for functions that convert external modules to JSON.